t/router/radixtree-method.t

```diff
@@ -16,9 +16,20 @@
 #
 use t::APISIX 'no_plan';
 
+our $yaml_config = <<_EOC_;
+apisix:
+    node_listen: 1984
+    router:
+        http: 'radixtree_uri'
```

Comment:
Can't this test pass with the default router?

Reply:
It can pass, should we use the default router here?

Reply:
Yes, it would be better. Adding special configuration here makes it like a special router is required.

Reply:
I was wondering if it would be better to specify the routing engine for these test cases on router, so that the default engine is changed and the test cases are not affected

Reply:
It's fine if you want to keep it.